src/middlewares/verifyAuth.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require("jsonwebtoken");
+const secret = process.env.JWT_SECRET;
+
+const verifyAuth = (req, res, next) => {
+  const token = req.headers.authorization.split(" ")[1];
```

Comment:
If `req.headers.authorization` is undefined then it will throw an error, please validate that.

`const token = req.headers.authorization.split(" ")[1];` Here token maybe undefined. Consider `Authorization: Bearer`

And for security reason, you should check `req.headers.authorization.split(" ")[0] === "Bearer" `

Reply:
Resolved.